src/Schema.php

```diff
@@ -61,6 +61,11 @@ protected function get_link($tagName)
                 unset($attrs['story'], $attrs['uuid']);
             }
 
+            if (isset($attrs['custom']) && is_array($attrs['custom'])) {
+                $attrs = array_merge($attrs, $attrs['custom']);
+            }
+            unset($attrs['custom']);
```

Comment:
Hi @uuf6429 , just a question for clarification, why this unset is not in the if above?

Reply:
as per the 3rd point:

> fix bug where **empty custom attributes array** (or non-array) would still errors because **isset($emptyArray) is false** and **unset() would not be called**